src/main/java/hu/flowacademy/zetaabsencemanager/service/AdminAbsenceService.java

```diff
@@ -107,4 +120,20 @@ public void delete(@NotNull Long id) {
     update(id, deleted);
   }
 
+  public Specification<Absence> getFilteredAbsences(Long administrationID, Type type,
+      Status status, User reporter, User assignee, LocalDate start, LocalDate end,
+      Integer dayStart, Integer dayEnd) {
+    Specification<Absence> spec = Specifications
```

Comment:
Szép, de ez így túl bonyolult. Ne legyen `FilterByType` és hasonló osztályok. A `Specifications.where()` tud lambdát is várni. Legyen egy lambda, amin belül az összes predikátum megvalósul.